std_msgs/Header header

float32 roll_setpoint
float32 pitch_setpoint
float32 yaw_setpoint
